bfdd/bfdd_vty.c

```diff
@@ -743,7 +743,7 @@ DEFPY(bfd_clear_peer_counters, bfd_clear_peer_counters_cmd,
 				ifname, vrfname);
 	if (bs == NULL)
 		return CMD_WARNING_CONFIG_FAILED;
-    
+
```

Comment:
are there any changes in this file ?. If not this could be removed from the patch